lua/lualine.lua

```diff
@@ -62,10 +64,14 @@ local function StatusLine(isFocused)
   if not isFocused then
     sections = M.inactiveSections
   end
-  if type(M.theme) == 'string' then
-    M.theme = utils.setTheme(M.theme)
+  if not theme_set  then
```

Comment:
A solution for fixing this bug while keeping functionality of live theme changing might be something like this
```lua
local theme_set = {}
M.theme = 'gruvbox'
--
--
if theme_set ~= M.theme then
  if type(M.theme) == 'string' then
        M.theme = utils.setTheme(M.theme)
   end
   highlight.createHighlightGroups(M.theme)
   theme_set = M.theme
```
Let me know what you think about this solution.